### DISCUSSION TITLE
Auto3dseg multi labels

### DISCUSSION THREAD

**Discussion by Malitha123** [Project-MONAI/tutorials#1655] [General]
I have a segmentation task with 3 labels. May I know how to segment it using Auto3Dseg? All the examples are for one label tasks.

**Comment by diazandr3s** [comment]
Hi @Malitha123,

I'd recommend you follow this example for multi-label segmentation tasks: https://github.com/Project-MONAI/tutorials/tree/main/auto3dseg/tasks/btcv

Which organs/tissues do you want to segment?

**Comment by Malitha123** [comment]
I am working on my own dataset for heart segmentation with consists of 5 labels (LA, RA, and so on). Those masks are in separate folders. For example Patient001 has 5 masks.

**Comment by diazandr3s** [comment]
Hi @Malitha123,

The first step should be merging the masks into a single file. Then create the YAML and JSON files to train Auto3DSeg.
Make sure the label indexes are in consecutive order and consistent throughout the dataset. i.e. LA is always 1, RA is 2, ...

Hope this helps,

**Comment by NastaranVB** [comment]
Hi @diazandr3s ,
In my segmentation task, there are 2 labels: infarction and hemorrhage parts in brain CT images. (3 labels when considering the brain as background). I would be thankful if you could also tell me which segmentation example fits my task more accurately.

**Comment by diazandr3s** [comment]
Hi @NastaranVB,

Are the segments overlapping? Have you seen this tutorial? https://github.com/Project-MONAI/tutorials/tree/main/auto3dseg/tasks/kits23

**Comment by NastaranVB** [comment]
Hi @diazandr3s,
Thanks for getting back in touch.
yes, they are overlapping.

**Comment by diazandr3s** [comment]
Hi @NastaranVB,

Thanks!
I think this is the best example for this sort of task: https://github.com/Project-MONAI/tutorials/blob/main/auto3dseg/tasks/kits23/input.yaml

Hope this helps,
